### PR TITLE
[FEATURE: UIPT-5] - Setup router

### PR DESCRIPTION
**Pull request by VasyaAndrela**

### CODE REVIEW

File: src/app/components/app.js [hunk 1]
```diff
@@ -1,9 +1,32 @@
...
+import ProductList from './ProductList';
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
+
 
 export default class App extends Component {
   render() {
     return (
-      <h1>Hello Interns</h1>
+      <Router>
+        <div>
```
Comment (AlyonaPianykh) [inline src/app/components/app.js]: I'm pretty sure this div can be removed

File: src/app/components/app.js [hunk 2]
```diff
@@ -1,9 +1,32 @@
...
+          </Switch>
+        </div>
+      </Router>
     );
   }
 }
+
+function Home() {
+  return(
+  <div>
```
Comment (AlyonaPianykh) [inline src/app/components/app.js]: And this one

File: src/app/components/ProductList.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+function ProductList() {
+  return (
+    <div>
```
Comment (AlyonaPianykh) [inline src/app/components/ProductList.js]: and this div also can be removed

File: src/app/components/app.js (hunk 1 above)
Comment (VasyaAndrela) [reply src/app/components/app.js]: it's just testing

**Pull request closed without merging**